app/views/registration_wizard/choose_your_npq.html.erb

```diff
@@ -1,6 +1,12 @@
-<h1 class="govuk-heading-xl"><%= t("helpers.title.registration_wizard.course_identifier") %></h1>
+<h1 class="govuk-heading-xl"><%= t("helpers.title.registration_wizard.choose_your_npq") %></h1>
```

Comment:
This wasn't strictly necessary, but I wanted its key to match the hint key below and if I used `helpers.hint.registration_wizard.course_identifier_html` then the copy was inserted below "Which NPQ do you want to do?" automatically.